tests/run-pass/packed_struct.rs

```diff
@@ -0,0 +1,14 @@
+#[repr(packed)]
+struct S {
+    a: i32,
+    b: i64,
+}
+
+fn main() {
+    let x = S {
+        a: 42,
+        b: 99,
+    };
+    assert_eq!(x.a, 42);
+    assert_eq!(x.b, 99);
```

Comment:
This test seems good, but can you add a `compile-fail` test that instead takes `let p = &x.b; *p;` to make sure it fails the alignment check on deref?